.github/workflows/plugins.yml

```diff
@@ -1147,7 +1156,7 @@ jobs:
       - run: yarn install
       - uses: ./.github/actions/node/oldest
       - run: yarn test:plugins:ci
-      - uses: ./.github/actions/node/16
+      - uses: ./.github/actions/node/latest
```

Comment:
bit weird to change 16 to latest when everywhere else in the PR you're changing 16 to oldest/18. Is this intended ?

Reply:
We went from testing only 16 and older to testing `oldest` to `latest`. I know restify has a focus on legacy, but as far as I'm aware it _should_ support all the way up to latest so I think it was just an oversight that the range before this change wasn't testing up to latest.